packages/cssnano-preset-default/src/__tests__/css-declaration-sorter.js

```diff
@@ -34,9 +34,23 @@ test(
 );
 
 test(
-  'keep order of vendor prefixed properties, mixed with short- and longhand',
-  passthroughCSS(
-    'a{animation-name:a;animation:b;-webkit-animation:c;-webkit-animation-name:d}'
+  'keep order of border properties, short- and longhand, left to right',
+  passthroughCSS('a{border-color:purple;border-top:1px solid}')
```

Comment:
I also had trouble with this test example, if you reduce it to the essential it fails because of some non-optimal `merge-lonhand` behaviour. I suggest you replace it with the exact example in the [original bug report](https://github.com/cssnano/cssnano/issues/1054): 
```suggestion
  processCSS(
    'a{border-color:purple;border-top-width:1px;border-top-style:solid}',
    'a{border-top:1px solid;border-color:purple}'
  )
```